src/libc/mod.rs

```diff
@@ -80,6 +83,86 @@ pub unsafe fn strlen(cs: *const c_char) -> size_t {
     count
 }
 
+pub struct EnvVal {
+    pub key: Cow<'static, [u8]>,
+    pub value: Cow<'static, [u8]>,
+}
+
+#[allow(unused)]
```

Comment:
Hmm, why is the `allow(unused)` required? `ENV_PTR` is used in `_init` down below, no? (Same with AUXVAL)